pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -27,6 +27,7 @@
 , tpmSupport ? true
 , uringSupport ? stdenv.isLinux, liburing
 , canokeySupport ? false, canokey-qemu
+, dbusDisplaySupport ? stdenv.isLinux
```

Comment:
Why only on Linux?

Reply:
Good question! I assumed dbus was mostly a Linux-specific thing, though maybe it isn't.

Just tried to build on aarch64-darwin:
```sh
env NIX_PATH=nixpkgs=https://github.com/bergkvist/nixpkgs/archive/qemu-dbus-display.tar.gz \
nix-shell -p "qemu.override { dbusDisplaySupport = true; }"
```

Though it seems to require some extra libraries (`epoxy/egl` and `gbm`)
```
../meson.build:1673:0: ERROR: Feature dbus_display cannot be enabled: -display dbus requires epoxy/egl and gbm

A full log can be found at /private/tmp/nix-build-qemu-7.1.0.drv-0/qemu-7.1.0/build/meson-logs/meson-log.txt

ERROR: meson setup failed

error: builder for '/nix/store/sxjg4blcvzi8l16fcvxwmj1y9d4k7gkg-qemu-7.1.0.drv' failed with exit code 1;
       last 10 log lines:
       > Run-time dependency fuse3 found: NO (tried pkgconfig)
       > Run-time dependency libbpf found: NO (tried pkgconfig)
       > Has header "IOKit/storage/IOMedia.h" : YES
       >
       > ../meson.build:1673:0: ERROR: Feature dbus_display cannot be enabled: -display dbus requires epoxy/egl and gbm
       >
       > A full log can be found at /private/tmp/nix-build-qemu-7.1.0.drv-0/qemu-7.1.0/build/meson-logs/meson-log.txt
       >
       > ERROR: meson setup failed
       >
       For full logs, run 'nix log /nix/store/sxjg4blcvzi8l16fcvxwmj1y9d4k7gkg-qemu-7.1.0.drv'.
```

Reply:
How about we leave it to linux for now and someone else can fix it for darwin later?

Reply:
As the maintainer of the package I'd really rather not see platform-specific functionality differences where they're not absolutely necessary.

If it absolutely must be disabled on Darwin, it should be `!stdenv.isDarwin` rather than `stdenv.isLinux`, because there's no reason to disable it for other platforms (e.g. BSD) just because it's broken on Darwin.  It should also have a comment explaining that it's not fundamentally broken on Darwin, just broken at present.